NOTICE.txt

```diff
@@ -422,8 +422,7 @@ This product includes software developed at CoreOS, Inc.
 
 --------------------------------------------------------------------
 Dependency: github.com/coreos/go-systemd
-Version: v18
```

Comment:
Just a reminder so we don't forget, try to continue using a tagged version.

Reply:
Bah, thought I did. Must have messed up govendor again.